docs/reference/templates.rst

```diff
@@ -194,11 +194,7 @@ can specify the templates to use in the ``Admin`` service definition:
 .. note::
 
     A ``setTemplates(array $templates)`` (notice the plural) method also
-    exists, that allows you to set multiple templates at once. Notice that,
-    if used outside of the service definition context,
-    ``setTemplates(array $templates)`` will replace the whole template list
-    for that ``Admin`` class, meaning you have to explicitly pass the full
-    template list in the ``$templates`` argument.
+    exists, that allows you to set multiple templates at once.
```

Comment:
We are changing override by merge. IMO will be nice to add note about it for people who know about previus functionality.

Reply:
I added a note in the upgrade note